package.json

```diff
@@ -21,19 +21,13 @@
     "@auto-it/all-contributors": "^10.37.6",
     "@auto-it/first-time-contributor": "^10.37.6",
     "@auto-it/slack": "^10.37.6",
-    "@grafana/eslint-config": "^2.5.0",
+    "@grafana/eslint-config": "^5.1.0",
     "@grafana/tsconfig": "^1.2.0-rc1",
     "@swc/core": "^1.2.162",
     "@swc/jest": "^0.2.20",
     "@types/jest": "^27.4.1",
     "@types/node": "18.11.7",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
```

Comment:
Are you sure we don't need these dependencies anymore? I can see them as part of the plugins common package.json template.

Reply:
I'm pretty sure they aren't needed as the dependencies in the package.json template are for after a plugin is scaffolded where as this worktree `package.json` has the common devDependencies for the workspaces `create-plugin` and `sign-plugin`. 

This fixes the node 18 install issue and cleans up as `@grafana/eslint-config` has most of these as nested deps too.

Reply:
Yea, as Jack says those are the dependencies of the scaffolded plugins, while these are the dependencies of the tools themselves. I think we could clean up the deps of the scaffolded plugins as well, but that's a separate step.